### ISSUE TITLE
Unable to open Pieces Core Services

### ISSUE THREAD

**Issue by martinafrazzoni** [pieces-app/support#546]
### Software

Desktop Application

### Operating System / Platform

Windows

### Your Pieces OS Version

11.0.2

### Early Access Program

- [ ] Yes, this is related to an Early Access Program feature.

### Kindly describe the bug and include as much detail as possible on what you were doing so we can reproduce the bug.

i am running into this issue after installing the new Pieces version. 
![Image](https://github.com/user-attachments/assets/7b14dbce-4947-4386-870b-c6953c4acb14)

**Comment by mark-at-pieces** [comment]
Hey there @martinafrazzoni,

Thank you for reaching out to support, our team is actively looking into this issue.

We will report as soon as we have more information, in the mean time if you uninstall and reinstall PiecesOS using a different installation method that would be awesome!

Thank you for your patience!

**Comment by mark-at-pieces** [comment]
may also be a duplicate of this issue, https://github.com/pieces-app/support/issues/545

not a bad idea to start here as well,

However I will keep this ticket open and report back when we have more information!

**Comment by Khuzaim123** [comment]
i uninstall and reinstall pieces os using different method but it still give the same error and not working

**Comment by mack-at-pieces** [comment]
@martinafrazzoni @Khuzaim123 we have identified the cause of this issue and will have a patch within the next week or so.

The issue arises when we can accurately detect the installation of PiecesOS, however PiecesOS fails to launch within a 10-second window.

We have addressed this by expanding the timeout window and handling the timeout case better. Linked to: https://github.com/pieces-app/pieces_for_x/issues/1119
